app/src/main/java/com/github/sdp/tarjetakuna/ui/webapi/WebApiViewModel.kt

```diff
@@ -106,7 +171,22 @@ open class WebApiViewModel : ViewModel() {
         }
         // TODO else get from cache
         else {
-            Toast.makeText(context, "No network available", Toast.LENGTH_SHORT).show()
+            Toast.makeText(
+                context,
+                "No network available, retrieving from cache",
```

Comment:
The fact this toast is copy/pasted many times indicates there's a need for some common function to do the operation